hiroaki-core/src/main/java/me/jorgecastillo/hiroaki/internal/MockServerRule.kt

```diff
@@ -13,7 +12,6 @@ class MockServerRule @JvmOverloads constructor(
 
     lateinit var server: MockWebServer
 
-    @Before
```

Comment:
This was accidental and never needed/used.

Reply:
Oops. Good catch!